deps/build.jl

```diff
@@ -43,3 +44,60 @@ end
 compile(TZDATA_DIR, COMPILED_DIR)
 
 info("Successfully processed TimeZone data")
+
+
+isdir(WINDOWS_DIR) || mkdir(WINDOWS_DIR)
+
+# Windows is weird and uses its own timezone
+info("Downloading Windows to TZ data")
+
+# Generate the mapping between MS Windows timezone names and
+# tzdata/Olsen timezone names, by retrieving a file.
+if 1==1
+    xml_source = "http://unicode.org/cldr/data/common/supplemental/windowsZones.xml"
+    xml_file = joinpath(WINDOWS_DIR, "windowsZones.xml")
+    remaining = 3
+
+    while remaining > 0
+        try
+            # Note the destination file will be overwritten upon success.
+            download(xml_source, xml_file)
+            remaining = 0
+        catch e
+            if isa(e, ErrorException)
+                if remaining > 0
+                    remaining -= 1
+                elseif isfile(xml_file)
+                    warn("Falling back to old region file $xml_file. Unable to download: $xml_source")
+                else
+                    error("Missing region file $region. Unable to download: $xml_source")
+                end
+            else
+                rethrow()
+            end
+        end
```

Comment:
You shouldn't need all this logic for downloading this file. It is only needed above to deal with the low availability IANA FTP server.